frontend/vue/components/UserAccount/ClassroomSection.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <section class="classroom">
+    <div class="classroom__title-container">
+      <h1 class="classroom__title">
+        {{ $translate("Classroom") }}
+      </h1>
+      <p class="classroom__title-text">
```

Comment:
Same comment as `SyllabusCreateLayout`

To me, this class is confusing because the title is text too. Maybe `syllabus__title__subtitle` or `syllabus__title__description` could be more descriptive. What do you think?

Reply:
updated w/ new component 👍🏽